### PR TITLE
clarify organizational account type requirements

### PR DESCRIPTION
**Pull request by jgravois**
'User' and 'Viewer' type accounts are not permitted to create hosted feature services.  They can upload a .zip file, but no additional options appear.  

Because of this we should definitely mention the explicit requirements in our own doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The process involves three steps:
 
 Here's how to get started:
 
-1. **Sign up for a free** [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/) - Your subscription allows you to download software, register apps, import data, create new services, store maps, perform spatial analysis and more. If you already have an [ArcGIS Organizational subscription](http://www.arcgis.com/features/plans/pricing.html) you can use that too, but please note that [free public accounts](https://geonet.esri.com/groups/geodev/blog/2014/10/15/help-i-cant-get-into-developersarcgiscom) will not work.
+1. **Sign up for a free** [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/) - Your subscription allows you to download software, register apps, import data, create new services, store maps, perform spatial analysis and more. If you already have an [ArcGIS Organizational subscription](http://www.arcgis.com/features/plans/pricing.html) (with at least publisher level permission) you can use that too. Please note that [free public accounts](https://geonet.esri.com/groups/geodev/blog/2014/10/15/help-i-cant-get-into-developersarcgiscom) and Organizational 'User" accounts will not work.
```
Comment (nixta) [inline README.md]: You have a quote mismatch around the word `User` near the end there.

**Comment by andygup** [comment]
Whoa, didn't realize how long this one had been sitting around...

**Comment by jgravois** [comment]
i addressed @nixta's entirely valid feedback and squashed.

**Comment by nixta** [comment]
Oh yucky. What editor are you using that adds whitespace at the ends of lines, @jgravois?

Makes it really hard to see what's actually changed 😫 I know that I can go to the URL and add `?w=1` to hide the whitespace differences but it'll still show up as diffs by default in the history. Any chance you could undo those and squash again please (he says stepping precariously on the demarcation of "entirely valid feedback")?

**Comment by jgravois** [comment]
my changes aren't adding whitespace, they're removing what was introduced by whatever text editor @alaframboise used in 87bf12728e0d2b1bed35125ab24c3ad17fcaad38.

tbh i can't see myself putting more work into this PR at this point :bowtie:.

**Comment by nixta** [comment]
Fair enough. My bad. I should have looked more closely at the changes. The diff between your prev commit and this threw me, but the additional change however it happened is +ve, I suppose. I'll go through and remove other trailing spaces for consistency if need be.

**Comment by jgravois** [comment]
> The diff between your prev commit and this threw me.

understandable.  it was cause i pulled from `master`.

**Comment by nixta** [comment]
Wait. I'm really confused. a) I don't see whitespace added in that commit you mention and b) why is it so different to what you submitted originally?

I'm also now struggling to merge with changes I've made in a restructuring branch (https://github.com/nixta/geodev-hackerlabs/tree/js4-update-and-develop-restructure) even though none of the changes conflict with the account type change you made. But as far as I can tell, none of them conflict with any changes (whitespace or otherwise), so git is really killing me on this one now.

If you have a moment tomorrow I'd love some help resolving that and making a PR of my changes.

**Comment by jgravois** [comment]
a) its not highlighted, but it must be at the trailing end of al's line.

b) i'm really not sure.  maybe i submitted this PR from a stale branch and didn't get it in sync until i incorporated your feedback the other day?  i don't see how though, and i can't trace the history because i blew it away ¯_(ツ)_/¯

[`nixta: js4-update-and-develop-restructure`](https://github.com/nixta/geodev-hackerlabs/tree/js4-update-and-develop-restructure)

> This branch is 17 commits ahead, 2 commits behind Esri:gh-pages

i'm happy to help you tomorrow.  you need to either _pull_ or _fetch/rebase_ the two commits introduced by this PR (my own and your merge commit) into your own branch. afterward we'll be able to pull your own work in cleanly.

i'm _pretty sure_ the fact that you tweaked line `68` and i added a new line `69` is what git is considering a conflict.  because of this, you need to compare **all** discrepancies between your own README.md and the remote README.md and choose to keep everything from your branch and incorporate my own one liner.  no wonder you were being so fussy about that whitespace!

**Comment by nixta** [comment]
Cool. Thanks. I did try merging the gh-pages into my branch but during the merge the repo got into a state I wasn't entirely confident of so it'll be good to have your adult supervision tomorrow :) I think I'm one git command and a sane review away from it being good.

**Pull request merged**